### DISCUSSION TITLE
Allow for global cameras + RTSP support.

### DISCUSSION THREAD

**Discussion by doucettom** [OctoFarm/OctoFarm#260] [Feature Request]
Was already discussed but can't find it here so just added it.

**Comment by NotExpectedYet** [comment]
Aye something I'm researching at minute, seems ffmpeg will become required if I go this route. Certainly want it myself, can skip OctoPrint then.

**Reply by doucettom** [reply]
I'd still use webcams on the Pis to individually monitor the prints but would put a Wyze Cam further back to get an eye on whole print farm

**Reply by NotExpectedYet** [reply]
Aye following. I was thinking about multiple camera streams on the printer originally so glad you said something. 

The trouble with RTSP is it doesn't display in web without been transcoded. I will have to add an extra dependency for this which is fine but I'm just waiting to get the backend changes done to split the printers by a type/category field. So they will be like "OctoPrint, Web Camera, CNC, DWC2, Klipper". Then whatever you add all flows inline with that.

**Reply by NotExpectedYet** [reply]
Updated the title just to remind me aha
